src/mmw/js/src/core/modificationConfig.json

```diff
@@ -141,18 +141,18 @@
         "copyStyle": "deciduous_forest"
     },
     "crop_tillage_no": {
-        "name": "No Till Ag",
-        "summary": "Growing crops or pasture from year to year without tilling or otherwise disturbing the soil, thereby increasing water infiltration, organic matter retention, and nutrient cycling in the soil.",
+        "name": "No Till Agriculture",
+        "summary": "The purpose of this BMP is to leave some residue from harvested crops on the soil surface to reduce soil erosion. This practice is similar to the “Conservation Tillage” and “Reduced Tillage” practices, and only differs with respect to the amount of residue left on the ground. In this case, it is assumed that ground coverage with residual matter is at or greater than 60%.",
```

Comment:
Is it possible to break this text across several lines? It'd make commenting on the diff a little easier. 

I did want to make sure that for this and others the proper quote mark to use in the actual description text is `“` and not `"`.

Reply:
You can't merge strings (or do any other operations) in a JSON file, and adding line breaks within the string would just add extraneous spacing in the resulting text. I'm going to leave it as is.

Regarding the typographer's quotes, I was curious about them myself, but as they are working in all browsers I think we can leave them in.